### ISSUE TITLE
A question about the best performance of the test set

### ISSUE THREAD

**Issue by FelixHuiweiLin** [cauchyturing/UCR_Time_Series_Classification_Deep_Learning_Baseline#14]
[Time Series Classification from Scratch with Deep Neural Networks: A Strong Baseline] It can be seen from this paper that the author choose the best model that achieves the lowest training loss and report its performance on the test set. I wonder what if it overfits.
